### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by HoodieRocks**
These improvements consist of updating `requirements.txt`, fixing type errors, fixing other possible bugs and removing unused variables.

Nothing of concern. It should merge cleanly and without breaking changes

### CODE REVIEW

File: commands/mod.py
Comment (Silabear) [inline commands/mod.py:1]: Isn’t MAX meant to be capital because const?

File: events/highlighter/highlighter.py [hunk 1]
```diff
@@ -199,8 +199,10 @@ def switch_mode(mode):
 		return tokens
 
 	def highlight(func, theme="default"):
-		# Shotcuts
-		colors = Hl.Database.color_codes if theme == "default" else theme
+		# Shortcuts
+  
+		# the "shut up" type
+		colours: dict[str, str] = Hl.Database.color_codes # if theme == "default" else theme
```
Comment (Sleet827) [inline events/highlighter/highlighter.py:205]: W variable rename from fake English to real English, respect.

File: events/highlighter/highlighter.py
Comment (Silabear) [inline events/highlighter/highlighter.py:1]: I assume this is the latest highlighter :P

**Review by Silabear — APPROVED** [review]
Looks good bestie

File: events/highlighter/highlighter.py (hunk 1 above)
Comment (Silabear) [reply events/highlighter/highlighter.py:205]: Stop reviewing on my repositories, nerd

File: commands/mod.py
Comment (HoodieRocks) [reply commands/mod.py:1]: If it bothers you I'll change it back, but it wasn't defined where constants would usually be defined (top of file)

File: events/highlighter/highlighter.py
Comment (HoodieRocks) [reply events/highlighter/highlighter.py:1]: I think the merge conflicts would say otherwise

File: events/highlighter/highlighter.py (hunk 1 above)
Comment (Sleet827) [reply events/highlighter/highlighter.py:205]: > Stop reviewing on my repositories, nerd

It's an open-source project, it's meant to be reviewed. Also, I was just complimenting Cobblestone, just like I complimented you in a later commit for using the correct spelling.

**Comment by HoodieRocks** [comment]
You didnt want this anyways :3

**Comment by Silabear** [comment]
this was on my todo list

**Comment by HoodieRocks** [comment]
for 3 weeks?

**Comment by Silabear** [comment]
I forgot

**Pull request merged**